src/components/Navbar/Navbar.jsx

```diff
@@ -26,6 +36,12 @@ export function Navbar() {
             <i className="fas fa-heart"></i>
             <span className="icon-badge">0</span>
           </Link>
+          <button
+            onClick={() => setTheme(!theme)}
```

Comment:
just a nitpick try passing a function to update state.
because if the user clicks multiple times we may get a stale closure update